tests/legacy/generate_checkpoints.sh

```diff
@@ -47,10 +46,9 @@ done
 
 # use the PL installed in the environment if no PL version is specified
 if [[ -z "$@" ]]; then
-  pl_ver=$(python -c "import pytorch_lightning as pl; print(pl.__version__)")
```

Comment:
I use handle local to create one checkpoint with local codebase and I do not want to complicate logic in CI to determine versions

Reply:
And the issue is if we release for example, `2.1.0rc`, this will be void until we bump the version in main and all PR will be rebased/updated so if you want we can create both - version and local